src/main/java/frc/robot/commands/ShootCommandGroup.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class ShootCommandGroup extends SequentialCommandGroup {
+  /** Creates a new ShootCommandGroup. */
+  public ShootCommandGroup(ShooterSubsystem shooterSubsystem) {
+      addCommands(
```

Comment:
Since this is a complex sequence, definitely want to add documentation on what sequence is intended

Reply:
I'm expecting more of a high level description of the steps in the command group. 
e.g. Simultaneously aim and spin up the shooter. Then cycle the hopper to shoot all loaded power cells. 
^Not sure if that is the expected behavior, hence the need for comments...